core/pubnub_grant_token_api.c

```diff
@@ -1,4 +1,5 @@
 /* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
+#include "core/pubnub_memory_block.h"
```

Comment:
@Xavrax we have once `include` in header already.

Reply:
Yeah. I included it first in code file, then in header. Gonna fix it.